Sources/Moya/Plugins/AccessTokenPlugin.swift

```diff
@@ -6,47 +6,49 @@ import Result
 /// A protocol for controlling the behavior of `AccessTokenPlugin`.
 public protocol AccessTokenAuthorizable {
 
-    /// Represents the authorization header to use for requests.
+    /// Represents the authorization header or url parameter to use for requests.
     var authorizationType: AuthorizationType { get }
 }
 
 // MARK: - AuthorizationType
 
-/// An enum representing the header to use with an `AccessTokenPlugin`
+/// An enum representing the header and url parameter to use with an `AccessTokenPlugin`
 public enum AuthorizationType: String {
     case none
```

Comment:
Unrelated to this PR: Why is there a `none` case? It doesn't seem to be doing anything. Being a plugin, the `none` case should just be modeled by not using the plugin. Am I missing something?

Reply:
@AndrewSB Since `AccessTokenAuthorizable` is implemented by adding its conformance to an existing `TargetType`, there could be endpoints within that target that don't want auth

Reply:
Ahh, I see. Thanks for clearing that up 😄